### ISSUE TITLE
Help translate AirPodsDesktop

### ISSUE THREAD

**Issue by SpriteOvO** [SpriteOvO/AirPodsDesktop#18]
I hope *AirPodsDesktop* can be easily used by users in more languages, so I have implemented i18n in a few recent commits (mainly e3ada1e47dbe3af376adb34b8186f955569b878f), which allows more people to easily participate in l10n translations, even if they have no programming experience.

So if you want to translate this program into your language or want to improve an existing translation, please read the [Translation Guide](https://github.com/SpriteOvO/AirPodsDesktop/blob/main/CONTRIBUTING.md#-translation-guide) and then [open an issue](https://github.com/SpriteOvO/AirPodsDesktop/issues/new/choose) to tell us that you are willing or working on it. If you encounter any problems during this process, feel free to [open an issue](https://github.com/SpriteOvO/AirPodsDesktop/issues/new/choose) as well.

Since I am not a native English speaker, any grammar corrections or improvements are also welcome.

Thanks in advance for your contributions!

**Comment by tGecko** [comment]
Hey, I am willing to translate to German. Could you provide the .ts file for de_DE?

**Comment by SpriteOvO** [comment]
@tGecko Sure, I have created a new branch [l10n-de_DE](https://github.com/SpriteOvO/AirPodsDesktop/tree/l10n-de_DE) and added `de_DE` locale and `.ts` file in commit 2baed33cc943965b84b9c3149d8c420ae8c3ab98.

You can fork this repo and checkout to that branch to start the translation, open a PR when you're done. Thanks!

**Comment by gir489returns** [comment]
Do you need a Russian or Ukrainian translation? I speak both fluently.

**Comment by SpriteOvO** [comment]
> Do you need a Russian or Ukrainian translation? I speak both fluently.

@gir489returns Of course, tell me the locale name you need and I'll create a branch for you so you can get the translation file to get started.

**Comment by gir489returns** [comment]
> > Do you need a Russian or Ukrainian translation? I speak both fluently.
> 
> @gir489returns Of course, tell me the locale name you need and I'll create a branch for you so you can get the translation file to get started.

Russian is ru_RU and Ukrainian is uk_UA. I pulled these from the LTSC builds. I don't know if they're entirely accurate and I don't use them to test, so for now let's just file them under that and assume Microsoft put the right region encoding in their titles. Give me the translation sheet that needs to be translated.

**Comment by gir489returns** [comment]
I really fucking hate git and hate using it at work, can I skip making a pull request and all that shit and just post it here when I'm done? I use SVN tied to a Jenkins machine at work.

**Comment by gir489returns** [comment]
Ugh this is confusing. I will have a lot of feedback notes when we're done.... Hahahahaha.

**Comment by gir489returns** [comment]
OK here we go. Round 1. Russian translation is complete and English notes are applied to the comments section. Rule number 1 of UX:
DO NOT YELL AT THE USER! Telling the user words like oops, sorry etc insults their intelligence. The user vaguely knows what they want to do, but doesn't know how to do it. Simply give them instructions that is all. People hate it when computers try to act human. That us UX 101. I know in China they try to make computers more human friendly, and I can tell you just translated this from Mandarin to English with a translator and your basic knowledge of English, but some words you used were too complex to translate into other languages. They're in my notes if you want to ask me specific questions why I changed the words.

Here is the RU translation. Let me know when I can begin work on the Ukrainian translation.

[Russian Translation.txt](https://github.com/SpriteOvO/AirPodsDesktop/files/8420034/Russian.Translation.txt)

**Comment by SpriteOvO** [comment]
If you want to translate it into your language, please [open a new issue](https://github.com/SpriteOvO/AirPodsDesktop/issues/new/choose).
